### DISCUSSION TITLE
API's for deleting/ overwriting the App service Profile configuration  in Edgex3.1 version(Napa)

### DISCUSSION THREAD

**Discussion by Devendra-7** [edgexfoundry/community#284] [Q&A]
Hello All,

In the Edgex Napa version i.e 3.1 didn't find any API's/commands to delete/overwrite the existing configuration(for a specific profile ex: rule-engine / mqtt-export).
In the case of device-virtual service we can delete any device using API's
Ex:curl -X DELETE localhost:59881/api/v3/device/name/Modbus-RTU-test-device.

How to update the our app service profile configuration (ex; mqtt-export) when using Consul provider and without consul provider.
Thanks in Advance...!

**Comment by cloudxxx8** [comment]
app services cannot be deleted via API
if you want modify the configs from Consul, you can try the Consul API or UI
